### PR TITLE
fix: grammar mistakes

### PR DESCRIPTION
**Pull request by thisislvca**
## What does this PR do?

Ths PR fixes some grammar mistakes I saw while reading through appwrite's docs. There are more dotted around, but I just fixed the few I saw myself :)
## Test Plan

Not needed.

## Related PRs and Issues

None.
### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/docs/advanced/security/gdpr/+page.markdoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-- Appwrite users will retain access to their personal information including the right to correct and to delete it.
+- Appwrite users will retain access to their personal information including the right to correct and delete it.
 - Impose the same rules upon the organization’s sub-processors who assist in providing Appwrite’s services as described in the Terms of Service (“ToS”). 
 - Appwrite will notify users promptly about policy changes and/or data breaches.
 
 You can learn more in our [Privacy policy](https://appwrite.io/privacy) and [Cookie policy](https://appwrite.io/cookies). 
 You can also reach us at `privacy@appwrite.io` for more questions.
 
-Appwrite has also implemented the following security measures to achieve technical compliance.
+Appwrite has also implemented the following security measures to achieve technical compliance:
```
Comment (gewenyu99) [inline src/routes/docs/advanced/security/gdpr/+page.markdoc:18]: Please revert this

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Hey, thanks for the PR, there's just one thing, we don't use `:` colons before lists as a stylistic choice.

File: src/routes/docs/advanced/security/gdpr/+page.markdoc (hunk 1 above)
Comment (thisislvca) [reply src/routes/docs/advanced/security/gdpr/+page.markdoc:18]: Well, you do it at line 10 of that same file... :)

Comment (thisislvca) [reply src/routes/docs/advanced/security/gdpr/+page.markdoc:18]: But can remove both if you want

**Comment by thisislvca** [comment]
Yo boys I actually fixed it so now it should be ready to get merged :)

**Review by loks0n — APPROVED** [review]

**Comment by loks0n** [comment]
> Yo boys I actually fixed it so now it should be ready to get merged :)

Thanks for your contribution 🚀

**Pull request merged**